epochStart/mock/requestHandlerStub.go

```diff
@@ -13,15 +13,40 @@ type RequestHandlerStub struct {
 	RequestRewardTxHandlerCalled       func(destShardID uint32, txHashes [][]byte)
 	RequestMiniBlocksHandlerCalled     func(destShardID uint32, miniblockHashes [][]byte)
 	RequestStartOfEpochMetaBlockCalled func(epoch uint32)
+	SetNumPeersToQueryCalled           func(key string, intra int, cross int) error
+	GetNumPeersToQueryCalled           func(key string) (int, int, error)
 }
 
-// SetIntraAndCrossShardNumPeersToQuery -
-func (rhs *RequestHandlerStub) SetIntraAndCrossShardNumPeersToQuery(key string, intra int, cross int) error {
+// SetEpoch -
+func (rhs *RequestHandlerStub) SetEpoch(epoch uint32) {
```

Comment:
Same here, you could have made an interface for the RequestHandler with just the methods you need.

Reply:
made a smaller interface for the meta block processor, but I can't make it smaller because in process.go it is sent to external component which require full functionality (require process.RequestHandler)